test/client_helper_test.rb

```diff
@@ -202,4 +202,16 @@
       end
     end
   end
+
+  describe "v3 recaptcha" do
+    it "renders input" do
+      html = recaptcha_v3 action: :foo
+      html.must_include('<input type="hidden" name="g-recaptcha-response[foo]" id="g-recaptcha-response-foo" data-sitekey="0000000000000000000000000000000000000000" class="g-recaptcha g-recaptcha-response "/>')
+    end
+
+    it "not have obsole script tag" do
```

Comment:
```suggestion
    it "not have obsolete closing script tag" do
```

Reply:
it not have correct grammar :D

-> it does not have ;)